modules/serverless-connect-sink-source-odc.adoc

```diff
@@ -18,9 +18,9 @@ When you create an event source by using the {product-title} web console, you ca
 
 .Procedure
 
-. Create an event source of any type, by navigating to *+Add* -> *Event Sources* and then selecting the event source type that you want to create.
+. Create an event source of any type, by navigating to *+Add* -> *Event Source* and then selecting the event source type that you want to create.
 
-. In the *Sink* section of the *Event Sources* form view, select *Resource*. Then use the drop-down list to select your sink.
+. In the *Sink* section of the *Create Event Source* form view, select *Resource*. Then use the drop-down list to select your sink.
```

Comment:
I think this must be relatively new guidance because we had always previously used "drop-down" and it's used across all the serverless docs, but I will open a follow up Jira to update this in all our procedures.

It would be a bit out of scope to change this for this PR since the text wasn't added or modified for that, and it would also need to apply to a lot of other modules, and for versions as far back as 4.6 rather than just 4.8+, so I'll leave this for now but will follow it up as a batch change later.

Reply:
Opened a follow up Jira for this https://issues.redhat.com/browse/SRVCOM-1918